queries.sql

```diff
@@ -31,3 +31,43 @@ WHERE NAME <> 'Gabumon';
 SELECT *
 FROM animals
 WHERE weight_kg BETWEEN 10.4 AND 17.3;  
+
+
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+COMMIT;
+
+
+BEGIN;
+DELETE
+FROM animals;
+ROLLBACK;
+
+
+BEGIN;
+DELETE
+FROM animals
+WHERE date_of_birth > 'January 1, 2022';
+SAVEPOINT younger_deleted;
+UPDATE animals
+SET weight_kg = - (weight_kg);
+ROLLBACK TO younger_deleted;
+COMMIT;
+
+SELECT count(*) FROM animals;
+SELECT count(*) FROM animals
+WHERE escape_attempts > 0;
```

Comment:
- [ ] this query should show How many animals have never tried to escape, instead of how many animals have tried to escape.